Makefile

```diff
@@ -271,38 +279,47 @@ $(foreach prog,$(PROGRAMS),$(eval $(call BUILD_ASSEMBLY,$(prog))))
 
 ########################## MAKE/INSTALL RULES ##########################
 #
-default: dependencies core
+default: core
 
 core: game renderers mods utility
 
 tools: editor gamemonitor
 
-package: cli-dependencies core editor gamemonitor docs version
+package: all-dependencies core tools docs version
```

Comment:
This will pull down the linux dependencies when running the packaging scripts, which we don't want (each platform's packaging script should handle those themselves).